src/operator/custom/custom-inl.h

```diff
@@ -76,7 +77,8 @@ class CustomOperator {
             bool training, const std::vector<NDArray>& arrs,
             const std::vector<int>& tags,
             const std::unordered_set<int>& output_tags,
-            const std::vector<NDArray>& outputs) {
+            const std::vector<NDArray>& outputs,
+	          const std::string op_type = "") {
```

Comment:
nit: indentation

Reply:
oh gee, fixing it now

Reply:
is it possible that op_type can be empty ?(i.e. the user will not specify any name for his/her custom operator ?) Since this method is only invoked for custom operators I want to understand why do we need a default values unless user can specify empty string as name for operator

Reply:
op_type will not be empty. This function is called in three places: custom.cc Forward(), custom.cc Backward(), and in src/c_api/c_api_function.cc. We need to pass in op_type in the first two places. Having a default value helps avoid creating a new api.